renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "ignoreDeps": [],
+  "packageRules": [
+    {
+      "packagePatterns": ["*"],
+      "semanticCommitType": "chore"
+    },
+    {
+      "depTypeList": [
+        "dependencies"
+      ],
+      "semanticCommitType": "deps"
+    },
+    {
+      "packagePatterns": ["^io.grpc:grpc-"],
+      "groupName": "gRPC packages"
+    },
+    {
+      "packagePatterns": ["^com.google.protobuf:protobuf-"],
+      "groupName": "Protobuf packages"
+    },
+    {
+      "packagePatterns": ["^io.opencensus:opencensus-"],
+      "groupName": "OpenCensus packages"
+    }
```

Comment:
Should we include:
- gax & gax-grpc?
- google-cloud-core & google-cloud-core-grpc?
- auto-value  & auto-value-annotations

Reply:
Let's take this as a separate PR and discuss further (I'd like to keep this just as the generated stuff for now). I've opened #13 to track.